README.md

```diff
@@ -14,7 +14,7 @@
 
 [Nixpkgs](https://github.com/nixos/nixpkgs) is a collection of over
 80,000 software packages that can be installed with the
-[Nix](https://nixos.org/nix/) package manager. It also implements
+[Nix](https://search.nixos.org/packages) package manager. It also implements
```

Comment:
```suggestion
[Nix](https://github.com/nixos/nix) package manager. It also implements
```

I couldn't find a dedicated page for nix on nixos.org, I think the github repository might be better here than the search page

Reply:
The reason I chose the search page is because that's the shortcut I wanted myself, The first sentence already includes the github link + home page.

I'll have to make a new pr because this repo is too big to git clone lol.